fedmsg.d/sse.py

```diff
@@ -0,0 +1,12 @@
+config = {
+
+    # SSE
```

Comment:
Well, we're already in the ``sse.py`` file :)

It's just that the comment here could lead to think that it's related to the sse server, while it's related to how the sse server connects to rabbitmq. So maybe we can just reword a little the comment? :)